.drone/drone.yml

```diff
@@ -193,6 +193,40 @@ volumes:
     host:
       path: /var/run/docker.sock
 
+---
+kind: pipeline
+type: docker
+name: Windows-Containerize
+platform:
+  os: windows
+  arch: amd64
+  version: "1809"
+trigger:
+  ref:
+    - refs/heads/main
+    - refs/tags/v*
+    - refs/heads/dev.*
+steps:
+  - name: Build Containers
+    image: grafana/agent-build-image:0.19.0-windows
+    volumes:
+      - name: docker
+        path: //./pipe/docker_engine/
+    environment:
+      DOCKER_LOGIN:
+        from_secret: DOCKER_LOGIN
+      DOCKER_PASSWORD:
+        from_secret: DOCKER_PASSWORD
+    commands:
+      - 'Invoke-Expression "C:\''Program Files''\git\bin\bash.exe -c ''.\\\\tools\\\\ci\\\\docker-containers-windows''"'
```

Comment:
I am actively working on this new shell script and will push it when it is ready for review